app/src/main/res/values/strings.xml

```diff
@@ -350,10 +350,18 @@
     <string name="errorpage_httpsonly_message_summary">However, it’s also possible that an attacker is involved. If you continue to the website, you should not enter any sensitive info. If you continue, HTTPS-Only mode will be turned off temporarily for the site.</string>
     <!-- Preference for accessibility -->
     <string name="preferences_accessibility">Accessibility</string>
+    <!-- Title of preference for sync debugging (only shown in the when the secret debug menu is enabled) -->
+    <string name="preferences_sync_debug">Sync Debug</string>
     <!-- Preference to override the Firefox Account server -->
     <string name="preferences_override_fxa_server">Custom Firefox Account server</string>
     <!-- Preference to override the Sync token server -->
     <string name="preferences_override_sync_tokenserver">Custom Sync server</string>
+    <!-- Preference to override the Push server -->
+    <string name="preferences_override_push_server">Custom Push server</string>
```

Comment:
Do these strings need to be translated? `Custom Sync Server` are strings that meant for end-users, where as debug ones are usually meant for us (internal contributors).

If they are meant for end-users, our string descriptions should also include the bit about how to access these settings (as you've done above with "only shown in the when the secret debug menu is enabled"). Each string is individually translated by l10n contributors.

If not, we can move these to the [`static_strings.xml`](https://github.com/mozilla-mobile/fenix/blob/main/app/src/main/res/values/static_strings.xml) file.

Reply:
Good catch.  I moved the new strings to `static_strings.xml`.  I considered moving the related Fxa/Token server strings over too, but I noticed the had some translations and were maybe used for the China release, so I just left those alone.

Reply:
The Fxa/Token Server ones are translated because they are user facing strings for clients that host custom servers, so I think those make sense.